R/derive_date_vars.R

```diff
@@ -1263,10 +1263,15 @@ compute_tmf <- function(dtc,
 #'
 #' @param flag_imputation Whether the date imputation flag must also be derived.
 #'
-#'   If `"auto"` is specified, the date imputation flag is derived if the
-#'   `date_imputation` argument is not null.
+#'   If `"auto"` is specified and `date_imputation` argument is not null or ,
+#'   then date imputation flag is derived.
```

Comment:
```suggestion
#'   If `"auto"` is specified and `highest_imputation` argument is not `"n"`,
#'   then date imputation flag is derived.
```
I think we missed to update this when we introduced the `highest_imputation` argument.

Could you update the description in `derive_vars_dtm()` as well?

Reply:
updated as suggested